src/Files.App/ViewModels/Properties/Items/FileProperty.cs

```diff
@@ -363,9 +363,22 @@ public async static Task<List<FileProperty>> RetrieveAndInitializePropertiesAsyn
 			{ "AddISO" , input => $"ISO-{(UInt16)input}"},
 			{ "RoundDouble" , input => $"{Math.Round((double)input)}"},
 			{ "UnitMM" , input => $"{(double)input} mm"},
+			{ "FormatEncodingBitrate", FormatEncodingBitrate }
 		};
 
 		private static string TimeSpanToString(TimeSpan t)
 			=> t.Days > 0 ? (t.Days * 24 + t.Hours) + t.ToString("':'mm':'ss") : t.ToString("hh':'mm':'ss");
+
+		private static string FormatEncodingBitrate(object input)
+		{
+			// For cases when multiple files are selected and it has a string value
+			if (input.GetType() != typeof(uint))
+				return input?.ToString() ?? string.Empty;
+
+			var sizes = new string[] { "bps", "kbps", "mbps", "gbps" };
```

Comment:
```suggestion
			var sizes = new string[] { "bps", "kbps", "Mbps", "Gbps" };
```
Mega and Giga should be capitalized.